### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Onaivio** [community/community#68731] [GitHub Education]
### Hi everyone,

Hello, I'm Onaivi, a computer science student from Nigeria with three years of experience in full-stack development. I specialize in leveraging JavaScript frameworks to create seamless user experiences. Additionally, I'm passionate about product and graphics design.

**Comment by drdixit** [comment]
👋Hello @Onaivio

**Reply by Onaivio** [reply]
Hi @drdixit 👋 
What's up
